ovs/extensions/healthcheck/expose_to_cli.py

```diff
@@ -0,0 +1,324 @@
+            if module_name or method_name:
+                try:
+                    for option in data[module_name]:
+                        if method_name:
+                            if option['method_name'] == method_name:
+                                return {module_name: [option]}
+                        else:
+                            return {module_name: data[module_name]}
+                except KeyError:
+                    pass
+            else:
+                return data
+            return None
+
+        def build_cache(hours=2):
+            """
+            Builds the internal cache of the methods with expose_to_cli decorator. This cache expires within 2 hours
+            :return: a dict the found items
+            :rtype: dict
+            """
+            # Build cache
+            # Executed from lib, want to go to extensions/healthcheck
+            found_items = {'expires': (datetime.now() + timedelta(hours=hours)).strftime(time_format)}
+            path = start_path
+            for root, dirnames, filenames in os.walk(path):
+                for filename in filenames:
+                    if filename.endswith('.py') and filename != '__init__.py':
+                        name = filename.replace('.py', '')
+                        file_path = os.path.join(root, filename)
+                        # Import file
+                        mod = imp.load_source(name, file_path)
+                        for member in inspect.getmembers(mod):
+                            if inspect.isclass(member[1]) and member[1].__module__ == name and 'object' in [base.__name__ for base in member[1].__bases__]:
+                                for submember in inspect.getmembers(member[1]):
+                                    if hasattr(submember[1], 'module_name') and hasattr(submember[1], 'method_name'):
+                                        if not submember[1].module_name in found_items:
+                                            found_items[submember[1].module_name] = []
+                                        # noinspection PyUnresolvedReferences
+                                        found_items[submember[1].module_name].append(
+                                            {'method_name': submember[1].method_name,
+                                             'module_name': name,
+                                             'function': submember[1].__name__,
+                                             'class': member[1].__name__,
+                                             'location': file_path,
+                                             'version': version_id
+                                             })
+            # Write the dict to a temp file
+            with open(temp_file_path, 'w') as f2:
+                f2.write(str(found_items))
+            return found_items
+
+        try:
+            with open(temp_file_path, 'r') as f:
+                exposed_methods = ast.literal_eval(f.read())
```

Comment:
Can't you do a `json.load(s)` here? `eval` actually try to execute the code in it which might be dangerous